include/fiction/technology/sidb_defects.hpp

```diff
@@ -72,6 +72,12 @@ enum class sidb_defect_type : uint8_t
      * Residual material.
      */
     GUNK,
+    /**
+     * Ionized arsenic atom as mentioned in \"Electrostatic landscape of a Hydrogen-terminated Silicon Surface Probed by
+     * a Moveable Quantum Dot\" by T. R. Huff, T. Dienel, M. Rashidi, R. Achal, L. Livadaru, J. Croshaw, and R. A.
+     * Wolkow in ACS Nano in 2019.
+     */
+    ARSENIC,
```

Comment:
I believe we have a test file for the other defect types. Could you add this one there?

Reply:
good point